modules/auxiliary/admin/kerberos/get_ticket.rb

```diff
@@ -0,0 +1,286 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Auxiliary::Report
+  include Msf::Exploit::Remote::Kerberos::Client
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Kerberos TGT/TGS Ticket Requestor',
+        'Description' => %q{
+          This module requests TGT/TGS Kerberos tickets from the KDC
+        },
+        'Author' => [
+          'Christophe De La Fuente' # MSF module
+        ],
+        'License' => MSF_LICENSE,
+        'Notes' => {
+          'Stability' => [ CRASH_SAFE ],
+          'SideEffects' => [ ],
+          'Reliability' => [ ]
+        },
+        'Actions' => [
+          [ 'TGT', { 'Description' => 'Request a TGT' } ],
+          [ 'TGS', { 'Description' => 'Request a TGS' } ],
+        ],
```

Comment:
Would you mind expanding this a bit more so the action names are in the form VERB_SUBJECT, and then the acronyms are expanded?

```suggestion
          [ 'GET_TGT', { 'Description' => 'Request a Ticket-Granting-Ticket (TGT)' } ],
          [ 'GET_TGS', { 'Description' => 'Request a Ticket-Granting-Service (TGS)' } ],
        ],
```